src/aequitas/fairness.py

```diff
@@ -182,7 +182,6 @@ def get_group_attribute_fairness(self, group_value_df, fair_measures_requested=N
                                                                   fair_measures_requested)
         if group_attribute_df.empty:
             logging.error('get_group_attribute_fairness: no fairness measures requested found on input group_value_df columns')
-            exit(1)
```

Comment:
happy to see all these `exit`s removed 😸 

looks like it's ok, but do they now need to `return` or `raise ...`?

Reply:
just added the raises